01_fib/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := "1\n1\n2\n3\n5\n8\n13\n"
```

Comment:
You can make it readable like `1
1
2
3
5
8
13'

Reply:
Updated!